pkg/util/strategicpatch/patch_test.go

```diff
@@ -23,10 +23,76 @@ import (
 	"strings"
 	"testing"
 
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/api/v1"
+
 	"github.com/davecgh/go-spew/spew"
 	"github.com/ghodss/yaml"
 )
 
+func getPod() *v1.Pod {
+	return &v1.Pod{
+		TypeMeta: unversioned.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: v1.ObjectMeta{
+			Name:      "ToBeDeletedPod",
+			Namespace: "ns1",
+			OwnerReferences: []v1.OwnerReference{
+				{
+					Kind:       "ReplicationController",
+					Name:       "owner1",
+					UID:        "123",
+					APIVersion: "v1",
+				},
+				{
+					Kind:       "ReplicationController",
+					Name:       "owner2",
+					UID:        "124",
+					APIVersion: "v2",
+				},
+			},
+			Finalizers: []string{
+				"f1",
+				"f2",
+				"f3",
+			},
+		},
+	}
+}
+
+func TestChao2(t *testing.T) {
```

Comment:
Unconventional naming system ;)

(test looks useful-- could make it an example if you don't think it adds coverage)